common/xml/convert_and_merge_composable_tests/composable-interconnect-implicit-parent.golden.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<xml>
+  <pb_type name="parent">
+    <input name="ia1"/>
+    <input name="ia2"/>
+    <input name="ia3"/>
+    <output name="o0"/>
+    <output name="o1"/>
+    <pb_type name="childa">
+      <input name="i1"/>
+      <input name="i2"/>
+      <output name="o"/>
+    </pb_type>
+    <pb_type name="childb">
+      <input name="i"/>
+      <output name="o"/>
+    </pb_type>
+    <pb_type name="childc">
+      <input name="i"/>
+      <output name="o"/>
+    </pb_type>
+    <interconnect>
+      <direct input="parent.ia1" name="childa-i1" output="childa.i1"/>
+      <mux name="childa-input-i2" input="parent.ia2 parent.ia3" output="childa.i2">
+        <metadata>
+          <meta name="fasm_mux">
+            parent.ia2 : 
+            parent.ia3 : 
+        </meta>
+        </metadata>
+      </mux>
+      <direct input="childa.o" name="parent-o0" output="parent.o0"/>
+      <direct input="childa.o" name="childb-i" output="childb.i"/>
+      <mux name="childc-input" input="childa.o childb.o" output="childc.i">
+        <metadata>
+          <meta name="fasm_mux">
+            childa.o : 
+            childb.o : 
+        </meta>
+        </metadata>
+      </mux>
+      <mux name="output" input="childa.o childb.o childc.o" output="parent.o1">
+        <metadata>
+          <meta name="fasm_mux">
+            childa.o : 
```

Comment:
This looks weird, should just emit NULL rather than ""?

Reply:
Fixed.